x-pack/plugins/task_manager/server/task_store.ts

```diff
@@ -574,8 +575,11 @@ export class TaskStore {
     let allTasks = new Array<ConcreteTaskInstance>();
 
     for (const response of responses) {
-      if (response.status !== 200) {
-        const err = new Error(`Unexpected status code from taskStore::msearch: ${response.status}`);
+      if (response.status && response.status !== 200) {
```

Comment:
**nit:** It's interesting that `response.status` could be null. If ever that occurs, should we consider it an unexpected status code as well?

Reply:
That's what I'm thinking, sounds good!